client/pages/sections/protocols/protocol-conditions.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import conditions from '../../../constants/protocol-conditions';
+
+class Content extends Component {
+  render() {
+    return (
+      <Fragment>
+        <div className="purple-inset">
+          <ReactMarkdown className="condition-text">{conditions.anaesthesia}</ReactMarkdown>
+        </div>
+        <div className="purple-inset">
+          <ReactMarkdown className="condition-text">{conditions.generalAnaesthesia}</ReactMarkdown>
+        </div>
+        <div className="purple-inset">
+          <ReactMarkdown className="condition-text">{conditions.surgery}</ReactMarkdown>
+        </div>
+        <div className="purple-inset">
+          <ReactMarkdown className="condition-text">{conditions.administration}</ReactMarkdown>
+        </div>
+      </Fragment>
+    );
+  }
+}
+
+class ProtocolConditions extends Component {
+  render() {
+    return (
+      <div className="protocol-conditions">
+        <h2>Conditions related to all protocols.</h2>
+
+        <p>
+          Anaesthesia, surgery, substance administration and withdrawal of fluid conditions relate to each of these protocols.
+        </p>
+
+        {
+          this.props.pdf ? <Content />
+          : (
+            <details>
+              <summary>View conditions</summary>
+              <br />
```

Comment:
What's the `<br/>` doing? Seems peculiar.

Reply:
Copied the html from https://github.com/UKHomeOffice/asl-projects/blob/0339d1cdb2e69b0b78edc9b415786687ac2501bf/client/components/species-selector.js#L80-L88 and didn't want to break any existing styling by faffing with the css.